src/OrchardCore/OrchardCore.Search.AzureAI.Core/Models/AzureAISearchDefaultOptions.cs

```diff
@@ -123,14 +123,14 @@ public class AzureAISearchDefaultOptions
     public void SetConfigurationExists(bool configurationExists)
         => _configurationExists = configurationExists;
 
-    public bool IsConfigurationExists()
+    public bool DoesConfigurationExist()
```

Comment:
Naming convention? I'm not sure we should use "Does". Just an opinion.

Reply:
Oops, I though it's `IsConfigurationExists()`